target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-haze.dts

```diff
@@ -303,3 +310,39 @@
 
 	qcom,ath11k-calibration-variant = "prpl-Haze";
 };
+
+&blsp1_i2c3{
+	pinctrl-0 = <&i2c_0_pins>;
+	pinctrl-names = "default";
+	status = "ok";
+
+	led-controller@30 {
+		compatible = "ti,lp5562";
```

Comment:
How is the driver kmod included on this device?
I dont see it being added to the default packages